ktx-mapbox-maps/src/main/java/com/mapbox/mapboxsdk/plugins/maps/Maps.kt

```diff
@@ -0,0 +1,62 @@
+package com.mapbox.mapboxsdk.plugins.maps
+
+import android.graphics.RectF
+import com.mapbox.geojson.Feature
+import com.mapbox.mapboxsdk.geometry.LatLng
+import com.mapbox.mapboxsdk.geometry.LatLngBounds
+import com.mapbox.mapboxsdk.maps.MapboxMap
+import com.mapbox.mapboxsdk.style.expressions.Expression
+
+/**
+ * Queries the map for rendered features
+ *
+ * @param latLng      the location to query
+ * @param layerIds    optionally - only query these layers
+ * @return the list of feature
+ */
+inline fun MapboxMap.queryRenderedFeatures(latLng: LatLng,
+                                           vararg layerIds: String): List<Feature> {
+    return this.queryRenderedFeatures(latLng,  null, *layerIds)
+}
+
+/**
+ * Queries the map for rendered features
+ *
+ * @param latLng      the location to query
+ * @param filter      filters the returned features with an expression
+ * @param layerIds    optionally - only query these layers
+ * @return the list of feature
+ */
+inline fun MapboxMap.queryRenderedFeatures(latLng: LatLng,
+                                           filter: Expression?,
```

Comment:
yes, it's nullable upstream

Reply:
Ah, ok, understand that. 
Though if there is special extension without this filter, so the user will have two ways of doing exactly the same - `queryRenderedFeatures(latLng, null, ids)` and `queryRenderedFeatures(latLng, ids)`. IMO its a bit confusing. And passing non-nullable value to nullable is perfectly safe)

Reply:
This is to match what we expose upstream, in simple cases the developer won't need an expression filter. The api was exposed like this due the varargs needing to be at the end of the method. If there a Kotlin construct I can leverage to avoid duplicating the functions (eg. something as an optional)?

Reply:
No, at least I don't know it.